Rules/CommonScripts/GameStateBanners.as

```diff
@@ -0,0 +1,189 @@
+#define CLIENT_ONLY
+
+Vec2f bannerStart = Vec2f_zero;
+Vec2f bannerPos = Vec2f_zero;
+Vec2f bannerDest = Vec2f_zero;
```

Comment:
You don't need to set all of these to `Vec2f_zero`, they are set to something right before you set the banner type. Just define them without value.

Reply:
fixed

Reply:
fixed